planner/cascades/testdata/integration_suite_in.json

```diff
@@ -63,6 +63,15 @@
       "select t2.a, t2.b from t1 right join t2 on t1.a = t2.a where t1.a > 2 and t2.b > 200"
     ]
   },
+  {
+    "name": "TestDoubleRead",
+    "cases": [
+      "select a, b from t order by t.a",
+      "select a, b from t order by t.b",
+      "select a, b, c from t order by t.b, t.c",
+      "select a, b, c from t order by t.c, t.b"
```

Comment:
Please add some SQLs with index hints.

Reply:
Why do we need the hints? Cascades planner doesn't support hint now.